src/SignalR/common/Http.Connections/test/LongPollingTests.cs

```diff
@@ -116,5 +116,25 @@ public void CheckLongPollingTimeoutValue()
             var options = new HttpConnectionDispatcherOptions();
             Assert.Equal(options.LongPolling.PollTimeout, TimeSpan.FromSeconds(90));
         }
+
+        [Fact]
+        public async Task IncludesLongRunningHeader()
+        {
+            using (StartVerifiableLog())
+            {
+                var pair = DuplexPipe.CreateConnectionPair(PipeOptions.Default, PipeOptions.Default);
+                var connection = new DefaultConnectionContext("foo", pair.Transport, pair.Application);
```

Comment:
I thought I read somewhere `foo` was never good in public-facing code/tests but nbd.

Reply:
fubar isn't allowed generally. But I think foo or bar separately is fine.

Reply:
fubar isn't allowed generally. But I think foo or bar separately is fine.